en/docs/reference/config-catalog.md

```
@@ -4417,6 +4417,63 @@ mode = "HYBRID"
+    <section>
+        <div class="mb-config-options">
+            <div class="superfences-tabs">
+
+            <input name="35" type="checkbox" id="_tab_35">
+                <label class="tab-selector" for="_tab_35"><i class="icon fa fa-code"></i></label>
+                <div class="superfences-content">
+                    <div class="mb-config-example">
+<pre><code class="toml">[apim.publisher]
+use_legacy_async_parser=true
+</code></pre>
+                    </div>
```

markdownlint-cli2 (0.18.1)

Code block style
Expected: fenced; Actual: indented

(MD046, code-block-style)